experimental/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -32,15 +149,193 @@ describe("TinyliciousClient", () => {
                 map1: SharedMap,
             },
         };
+        const [container] = await TinyliciousClient.createContainer(containerConfig, schema);
+        await new Promise<void>((resolve, reject) => {
+            container.on("connected", () => {
+                resolve();
+            });
+        });
+        assert.notStrictEqual(container, undefined, "Container cannot be created");
```

Comment:
I think we should check this another way.  The API returns a promise with type `Container`, which idiomatically means it should never be `undefined` like is being checked here if the promise successfully resolves, and we would instead expect the promise to reject if anything went wrong.  So if we're going the not undefined route, we should instead check that the promise does not throw (is there an assert.something api for that?).

Reply:
good point! And you are right, there is an assert.doesNotReject() that will awaits the returned promise to complete then check that the promise is not rejected.